cabal.project

```diff
@@ -213,48 +214,12 @@ constraints:
   , bech32 == 1.1.3
 
   -- Cardano Node dependencies:
-  , any.cardano-api ==8.39.3.0
-  , any.cardano-binary ==1.7.1.0
-  , any.cardano-cli ==8.20.3.0
-  , any.cardano-crypto ==1.1.2
-  , any.cardano-crypto-class ==2.1.4.0
-  , any.cardano-crypto-praos ==2.1.2.0
-  , any.cardano-crypto-wrapper ==1.5.1.1
-  , any.cardano-data ==1.2.1.0
-  , any.cardano-ledger-allegra ==1.3.0.0
-  , any.cardano-ledger-alonzo ==1.6.0.0
-  , any.cardano-ledger-api ==1.8.0.0
-  , any.cardano-ledger-babbage ==1.6.0.0
-  , any.cardano-ledger-binary ==1.3.0.0
-  , any.cardano-ledger-byron ==1.0.0.4
-  , any.cardano-ledger-conway ==1.12.0.0
-  , any.cardano-ledger-core ==1.10.0.0
-  , any.cardano-ledger-mary ==1.5.0.0
-  , any.cardano-ledger-shelley ==1.9.0.0
-  , cardano-node +systemd -unexpected_thunks
-  , any.cardano-ping ==0.2.0.13
-  , any.cardano-prelude ==0.1.0.4
-  , any.cardano-protocol-tpraos ==1.1.0.0
-  , any.cardano-slotting ==0.1.2.0
-  , any.cardano-strict-containers ==0.1.3.0
-  , any.ouroboros-consensus ==0.16.0.0
-  , any.ouroboros-consensus-cardano ==0.14.1.0
-  , any.ouroboros-consensus-diffusion ==0.14.0.0
-  , any.ouroboros-consensus-protocol ==0.7.0.0
-  , any.ouroboros-network ==0.15.0.0
-  , any.ouroboros-network-api ==0.7.1.0
-  , any.ouroboros-network-framework ==0.13.0.0
-  , any.ouroboros-network-mock ==0.1.1.1
-  , any.ouroboros-network-protocols ==0.8.1.0
-  , any.ouroboros-network-testing ==0.6.0.1
-  , any.plutus-core ==1.21.0.0
-  , any.plutus-ledger-api ==1.21.0.0
-  , plutus-scripts-bench -defer-plugin-errors -unexpected_thunks
-  , any.plutus-tx ==1.21.0.0
   , any.io-classes ==1.4.1.0
+  , io-classes -asserts
   , any.io-classes-mtl ==0.1.1.0
-
-
+  , cardano-node == 8.9.11
```

Comment:
Should this have been [8.11.0](https://chap.intersectmbo.org/package/cardano-node-8.11.0/)?